python/etl/dbt.py

```diff
@@ -32,10 +36,12 @@ def build_image(self):
         time.sleep(5)  # The image is not immediately available to pull
         return img
 
-    def run_cmd(self, cmd):
+    def run_cmd(self, cmd, detach=True, logs=True):
+        if logs and not detach:
+            raise ValueError("Logs cannot be set to True while detach is false")
```

Comment:
The mixed case here will bug me :-)
```suggestion
            raise ValueError("Logs cannot be set to True while detach is False")
```